### PR TITLE
bpo-39815: add cached_property to all

### PR DESCRIPTION
**Pull request by hakancelikdev**
https://bugs.python.org/issue39815

Automerge-Triggered-By: @pablogsal

### CODE REVIEW

**Review by isidentical — APPROVED** [review]
LGTM

File: Lib/functools.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 __all__ = ['update_wrapper', 'wraps', 'WRAPPER_ASSIGNMENTS', 'WRAPPER_UPDATES',
            'total_ordering', 'cmp_to_key', 'lru_cache', 'reduce',
            'TopologicalSorter', 'CycleError',
-           'partial', 'partialmethod', 'singledispatch', 'singledispatchmethod']
+           'partial', 'partialmethod', 'singledispatch', 'singledispatchmethod', "cached_property"]
```
Comment (pablogsal) [inline Lib/functools.py]: Could you move this to the next line so the line remains no longer than ~80 chars?

**Review by pablogsal — CHANGES_REQUESTED** [review]
Thanks for the PR. Just a minor comment, otherwise LGTM

**Comment by hakancelik96** [comment]
I have made the requested changes; please review again

**Review by pablogsal — APPROVED** [review]

**Comment by pablogsal** [comment]
Thanks for the PR, @hakancelik96 !

**Comment by hakancelik96** [comment]
> Thanks for the PR, @hakancelik96 !

You are welcome, thanks for the review 💯

**Pull request merged**